### PR TITLE
remove `object-hash` and `lru_map`

### PR DESCRIPTION
**Pull request by chmelevskij**
* use `crypto-es` instead of `object-hash`
* remove `lru_map` as it's not used anymore

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]
* successfully flashed 2 different unsoldered FC's with 4.3.2 and zulu-cloud; 
* configured both FC with 4.3.2 and zulu-cloud successfully, all tabs excluding GPS, VTX-tables, AUX-Adjustments, Servos, and Tethered-logging.

**Pull request merged**
